image_proc/src/nodelets/resize.cpp

```diff
@@ -93,7 +93,7 @@ void ResizeNodelet::configCb(Config &config, uint32_t level)
 void ResizeNodelet::subscribe()
 {
   sub_info_ = nh_->subscribe("camera_info", 1, &ResizeNodelet::infoCb, this);
-  sub_image_ = nh_->subscribe("image", 1, &ResizeNodelet::imageCb, this);
+  sub_image_ = nh_->subscribe("in_image", 1, &ResizeNodelet::imageCb, this);
```

Comment:
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
@vrabaud 
This causes a lot of error in our system!!!!!!!!!!!!!!!!!!!

Reply:
I could also fix this in https://github.com/ros-perception/image_pipeline/pull/309, but I'd wait for some feedback from the maintainers.

Is there anything else broken, other than the nodelet manager I had to removed in https://github.com/ros-perception/image_pipeline/pull/309?

Reply:
IMO, we should revert this PR first to get 'no-bug' repo, then we can start debugging.

Reply:
+1 for just reverting and then clarifying which part is 'incomplete'